src/common/Requirements.ts

```diff
@@ -182,6 +225,9 @@ export class RequirementProcessor {
                                         ? Promise.resolve()
                                         : Promise.reject(new Error());
                                 }),
+                            skip: () =>
+                                requirement.skipped == true &&
+                                requirement.title,
```

Comment:
Not sure if I understand the logic here... why do we need the `&& requirement.title`?

Reply:
Yup.  Removed.  Thanks!